### PR TITLE
[WIP] V2 doc update

### PR DESCRIPTION
**Pull request by soyuka**
- [x] Operations configurations
- [x] Getting started
- [ ] Filters configuration update
- [ ] Serializations groups configuration update
- [ ] Controllers - review
- [ ] Change DunglasApiBundle to ApiPlatform

Ref: https://github.com/api-platform/core/pull/490

### CODE REVIEW

**Comment by teohhanhui** [comment]
See #40

Collaborate, perhaps? :smile:

**Comment by soyuka** [comment]
I should've gone through pending PR's, there is also https://github.com/api-platform/doc/pull/37

**Comment by Simperfit** [comment]
@soyuka  I will be happy to help ;) you can cherry pick what i've started and we can collaborate on this PR ;).

File: api-bundle/operations.md [hunk 1]
```diff
@@ -17,100 +17,178 @@ By default, the following operations are automatically enabled:
...
-# ...
-
-    resource.product:
-        parent:    "api.resource"
-        arguments: [ "AppBundle\Entity\Product" ]
-        calls:
-            -      [ "initItemOperations", [ [ ] ] ]
-            -      [ "initCollectionOperations", [ [ "@resource.product.collection_operation.get" ] ] ]
-        tags:      [ { name: "api.resource" } ]
+```php
```
Comment (dunglas) [inline api-bundle/operations.md]: I would make annotations the first one (as this is the preferred default value).

**Comment by dunglas** [comment]
Integrated in #37. Please open new PRs against this branch.

**Pull request closed without merging**